models/Activity/Agenda.ts

```diff
@@ -78,6 +84,23 @@ export class AgendaModel extends BiDataTable<Agenda, AgendaFilter>() {
     };
   }
 
+  async getOne(id: string) {
+    await super.getOne(id);
+
+    const { mentors, title, forum, 负责人手机号 } = this.currentOne;
+
+    this.currentRecommend = new SearchAgendaModel();
+
+    await this.currentRecommend!.getList({
+      负责人手机号,
+      mentors,
+      summary: title,
+      forum,
```

Comment:
```suggestion
  async getOne(id: string) {
    await super.getOne(id);

    const { title } = this.currentOne;

    this.currentRecommend = new SearchAgendaModel(this.appId, this.tableId);

    await this.currentRecommend!.getList({
      title: words  // 分词逻辑自己写
```
1. 因安全原因，本项目的中文字段仅用于提交给飞书 API 筛选，不会包含在返回数据中。
2. 分词 API：https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter